tests/hwc-val/tests/hwc/framework/common/HwcvalDebug.cpp

```diff
@@ -105,7 +86,7 @@ int Mutex::unlock() {
 
 bool Mutex::tryLock() {
   ALOGD_IF(MUTEX_CONDITION_DEBUG, "Testing mutex %p thread %u", this, gettid());
-  mspinlock.lock();
+  //mspinlock.lock();
```

Comment:
we should lock and immediately unlock ? I dont see where we have the counterpart unlock here

Reply:
Adding lock is the correct.  it is unlocked by Mutex::unlock
Thanks